src/main/java/uk/gov/companieshouse/psc/delta/consumer/PscDeltaConsumer.java

```diff
@@ -39,7 +38,6 @@ public PscDeltaConsumer(Logger logger, KafkaTemplate<String, Object> kafkaTempla
      */
     @RetryableTopic(attempts = "${pscs.delta.retry-attempts}",
             backoff = @Backoff(delayExpression = "${pscs.delta.backoff-delay}"),
-            fixedDelayTopicStrategy = FixedDelayStrategy.SINGLE_TOPIC,
```

Comment:
i think you should maybe be replacing this with SameIntervalTopicReuseStrategy

Reply:
Let me check. Find some information that is it not required anymore.

Reply:
in the RetryableTopic interface this enabled as default as shown below:
SameIntervalTopicReuseStrategy sameIntervalTopicReuseStrategy() default SameIntervalTopicReuseStrategy.SINGLE_TOPIC;

Reply:
Ah very good, thanks Jeph!